include/Lv2Evbuf.h

```diff
@@ -47,15 +47,16 @@ namespace lmms
 /**
    An abstract/opaque LV2 event buffer.
 */
-typedef struct LV2_Evbuf_Impl LV2_Evbuf;
+using LV2_Evbuf = struct LV2_Evbuf_Impl;
 
 /**
    An iterator over an LV2_Evbuf.
 */
-typedef struct {
+using LV2_Evbuf_Iterator = struct LV2_Evbuf_Iterator
```

Comment:
OK, I start getting it. This is a construct to allow writing `struct LV2_Evbuf_Iterator` instead of `LV2_Evbuf_Iterator` in C. But in C++, I think it's the same.